content/en/docs/contribute/style/style-guide.md

```diff
@@ -22,8 +22,11 @@ discussion.
 <!-- body -->
 
 {{< note >}}
-Kubernetes documentation uses [Blackfriday Markdown Renderer](https://github.com/russross/blackfriday) along with a few [Hugo Shortcodes](/docs/home/contribute/includes/) to support glossary entries, tabs,
-and representing feature state.
+Kubernetes documentation uses
+[Blackfriday Markdown Renderer](https://github.com/russross/blackfriday)
```

Comment:
nit: Docs are rendered with Hugo and Goldmark Markdown renderer (https://github.com/yuin/goldmark).
Hugo is configured to render with this parser, but with some adjustments for content that previously rendered with Blackfriday.

Reply:
Okay. I was only focusing on the links in the PR.
Will address this.